### PR TITLE
Design-Admin

### PR DESCRIPTION
**Pull request by saimanoharhm**
## Related Issue 

Closes: #81

### Describe the changes you've made

I have added screenshots for admin design

## Type of change

What sort of change have you made:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How Has This Been Tested?

Describe how have you verified the changes made

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly whereever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

## Screenshots

 Original           | Updated
 :--------------------: |:--------------------:
 original screenshot | updated screenshot |

### CODE REVIEW

**Comment by saimanoharhm** [comment]
@praveenscience I have added screenshots for it.  Please review it.

**Comment by praveenscience** [comment]
Looks good. Why have you used all temporary images, they might get discarded! Also, your PR has some conflicts. Can you please fix the conflicts?

**Comment by praveenscience** [comment]
Why did you remove the md file?

**Comment by saimanoharhm** [comment]
> Why did you remove the md file?

I have added images directly @praveenscience

**Comment by saimanoharhm** [comment]
@praveenscience Now I have added images permanently and I removed conflict as well. You can check it now.

**Comment by praveenscience** [comment]
@saimanoharhm Is there no md file now?

**Comment by saimanoharhm** [comment]
> @saimanoharhm Is there no md file now?

@praveenscience Now you can check it

File: design_admin.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+3. Company Management  
+   ![Company Management](https://user-images.githubusercontent.com/72253236/116418806-181cdd80-a85a-11eb-913f-13a0dd50ed43.png)
+4. Content - steps management  
+   ![Content Steps management](https://user-images.githubusercontent.com/72253236/116418808-18b57400-a85a-11eb-90ea-c5b53fe86eb3.png)
+5. Courses Management  
+   ![Courses Management](https://user-images.githubusercontent.com/72253236/116418811-18b57400-a85a-11eb-8fbf-9cb21f61b0d7.png)
+6. Document Management  
+   ![Document Management](https://user-images.githubusercontent.com/72253236/116418813-194e0a80-a85a-11eb-8313-ad5da61e15d6.png)
+7.Roles Management  
+   ![Roles Management](https://user-images.githubusercontent.com/72253236/116418816-19e6a100-a85a-11eb-8113-85723dbb12fd.png)
```
Comment (praveenscience) [inline design_admin.md:18]: Convert Temp images to perm images.

**Comment by saimanoharhm** [comment]
@praveenscience Now you can check it.

**Review by praveenscience — CHANGES_REQUESTED** [review]
Erm???
image

**Comment by saimanoharhm** [comment]
Screenshot (2334)

**Comment by saimanoharhm** [comment]
Screenshot (2335)

**Review by praveenscience — APPROVED** [review]

**Pull request merged**